src/ElectricPokemon.java

```diff
@@ -0,0 +1,32 @@
+public class ElectricPokemon extends Pokemon{
+
+    String pokemonType;
+    String food;
+
```

Comment:
Ik weet dat dit de opdracht was, maar vraag je goed af waar je welke variabele instantieert. Deze twee zullen waarschijnlijk voor alle pokemon gelden, dan is het beter om ze in de superklasse te instantiëren